.github/workflows/cli-integration.yml

```
@@ -19,7 +19,11 @@ jobs:
 
       - name: Build Docker image
         run: |
-          docker login -u "$DOCKER_LOGIN" --password-stdin <<<"$DOCKER_TOKEN"
+          if [ -n "$DOCKER_LOGIN$DOCKER_TOKEN" ]; then
+              docker login -u "$DOCKER_LOGIN" --password-stdin <<<"$DOCKER_TOKEN"
+          else
+              echo "Not login-in to docker since no credentials were provided, might hit limits below"
```

```suggestion
              echo "Not logging in to docker since no credentials were provided, might hit limits below"
```

---

<details>
<summary>that is a word on itself which AFAIK has nothing to do with the process to login ;-)</summary> 

```shell
❯ dict logging
3 definitions found

From The Collaborative International Dictionary of English v.0.48 [gcide]:

  Log \Log\, v. t. [imp. & p. p. {Logged}; p. pr. & vb. n.
     {Logging}.]
     1. (Naut.), To enter in a ship's log book; as, to log the
        miles run. --J. F. Cooper.
        [1913 Webster]
  
     2. To record any event in a logbook, especially an event
        relating to the operation of a machine or device.
        [PJC]

From The Collaborative International Dictionary of English v.0.48 [gcide]:

  Logging \Log"ging\, n.
     The business of felling trees, cutting them into logs, and
     transporting the logs to sawmills or to market.
     [1913 Webster]

From WordNet (r) 3.0 (2006) [wn]:

  logging
      n 1: the work of cutting down trees for timber

```
</details>

ok, what about 

```suggestion
              echo "No credentials for docker were provided, no 'docker login' used."
```

?

---

"Logging in" is the progressive form of "to log in": https://www.merriam-webster.com/dictionary/log%20in

---

thanks for digging, let's then go with

```suggestion
              echo "Not logging in to docker since no credentials were provided, might hit limits below"
```